src/js/components/TabPanesComponent.jsx

```diff
@@ -56,6 +56,12 @@ var TabPanesComponent = React.createClass({
               className="btn btn-success pull-right">
                   + New App
             </Link>
+            <Link
+                to={path}
+                query={{modal: "chaos"}}
+                className="btn btn-default pull-right">
+                  Chaos
```

Comment:
See my comment below – with the current state of things we can't perform any check.